SchedulerJobs/SchedulerJobs.Services/SchedulerJobs.Services.csproj

```diff
@@ -13,7 +13,7 @@
 		<PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" />
 		<PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
 		<PackageReference Include="UserApi.Client" Version="1.49.2" />
-		<PackageReference Include="VideoApi.Client" Version="2.2.5" />
+		<PackageReference Include="VideoApi.Client" Version="3.0.2-pr-0691-0002" />
```

Comment:
Will update to latest once https://github.com/hmcts/vh-video-api/pull/691 merged